help.php

```diff
@@ -0,0 +1,124 @@
+                <li>and others</li>
+            </ul>
+        </li>
+        <li>Query Types: Shows to which upstream DNS the permitted requests have been forwarded to.</li>
+        <li>Top Domains: Ranking of requested sites by number of DNS lookups.</li>
+        <li>Top Advertisers: Ranking of requested advertisements by number of DNS lookups.</li>
+        <li>Top Clients: Ranking of total DNS requests separated by clients on the local network.</li>
+    </ul>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Query Log</h2>
+    <p>Shows the recent queries after parsing the pi-hole log files. It is possible to search through the whole list by using the "Search" input field. If the status is reported as "OK", then the DNS request has been permitted. Otherwise ("Pi-holed") it has been blocked. By clicking on the buttons under "Action" the corresponding domains can quickly be added to the white-/blacklist. The status of this action will be reported on this page.</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>White- / Blacklist</h2>
+    <p>Add or remove domains (or subdomains) from the white-/blacklist. If a domain is added to e.g. the whitelist, any possible entry of the same domain will be automatically removed from the blacklist and vice versa. Adding wildcards using the web UI is currently <em>not</em> supported.</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Update Lists</h2>
+    <p>Runs the command <pre>sudo pihole -g</pre> and prints the result transparently to the web UI. The gravity.sh script will update the list of ad-serving domains</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Disable / Enable</h2>
+    Disables resp. enables Pi-Hole DNS Blocking completely. The change will be reflected by a changed status (top left)
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Donate</h2>
+    If you like Pi-Hole, please consider a small donation. Keep in mind that Pi-hole is free, but powered by your donations
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Help (this page)</h2>
+    Shows information about what is happening behind the scenes and what can be done with this web user interface (web UI)
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Footer</h2>
+    Shows the currently installed Pi-hole and Web Interface version. If an update is available, this will be indicated here
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Emergency help</h2>
+    In case the web UI does not work properly anymore (i.e. timeout errors or diagrams not showing up) you can try to flush the Pi-hole config file by clicking <a href="#" id="flush">FLUSH</a>. Note that your statistics will be reset and you loose the statistics of the day until now.
```

Comment:
loose -> lose and "today's statistics up to this point. ~~the statistics of the day until now.~~"